augur/application.py

```diff
@@ -39,7 +39,9 @@ def __init__(self, given_config={}, disable_logs=False, offline_mode=False):
         self.gunicorn_options = {
             'bind': '%s:%s' % (self.config.get_value("Server", "host"), self.config.get_value("Server", "port")),
             'workers': int(self.config.get_value('Server', 'workers')),
-            'timeout': int(self.config.get_value('Server', 'timeout'))
```

Comment:
@ABrain7710 : This is the section of the code with the merge conflict. @sbrennan98 has a useful comment when you look at the merge conflict, and I think its these conditions where the install script may need to be modified. If we could script generating a certbot certificate (I'm not sure we can, or that its worth the labor), then we may want to use config variables for HTTP or HTTPS.